src/screens/Dashboard/TransactionList.tsx

```diff
@@ -54,20 +54,35 @@ export class TransactionList extends PureComponent<Props> {
     );
   };
 
-  render() {
+  renderTransactions = () => {
+    const sections = this.getSectionData();
+    return sections.map(section => {
+      const { data: transactions, title } = section;
+      return (
+        <View key={title}>
+          {this.renderSectionTitle(title)}
+          {transactions.map(t => (
+            <TransactionItem key={t.txid} item={t} onPress={this.onTransactionItemPress} />
```

Comment:
I don't get this change. A Virtualized list (SectionList / FlatList ) is necessary for lists with more items like TransactionList. The same mechanism is in ContactList or RecoveryTransactionListScreen. Rendering transactions in simple view is an antipattern.

Reply:
SectionList is nested here inside scrollView, so there is no performance gain here. https://nyxo.app/fixing-virtualizedlists-should-never-be-nested-inside-plain-scrollviews 
<img width="1055" alt="Zrzut ekranu 2020-08-19 o 09 31 47" src="https://user-images.githubusercontent.com/67273035/90605558-d2097780-e1fe-11ea-940e-b459242d535a.png">

Reply:
Great that you have found it. Please set ScreenTemplate noScroll prop to true here so SectionList is wrapped with View instead ScrollView.

Reply:
I tried this but unfortunately, it won't work as we weed scrollView to scroll down as the whole view can't fit in the size of the phone screen

Reply:
Have you tried to move everything above the TransactionList to its ListHeaderComponent prop?

Reply:
I didn't, seemed to be complicated to move the whole logic inside the header but I'll try to do it